ports/stm32l4/Makefile

```diff
@@ -0,0 +1,62 @@
+UF2_FAMILY_ID = 0x00ff6919
+CROSS_COMPILE = arm-none-eabi-
+
+ST_HAL_DRIVER = lib/st/stm32l4xx_hal_driver
+ST_CMSIS = lib/st/cmsis_device_l4
+CMSIS_5 = lib/CMSIS_5
+
+# List of git submodules that is included as part of the UF2 version
+GIT_SUBMODULES = st/cmsis_device_l4 st/stm32f4xx_hal_driver tinyusb

```

Comment:
st/stm32f4xx_hal_driver --> st/stm32l4xx_hal_driver

Reply:
Thanks, I hadn't seen this :-)